src/App/Pages/Vault/ViewPageViewModel.cs

```diff
@@ -764,6 +765,19 @@ public string ValueText
             }
         }
 
+        public string ValueAccessibilityText
+        {
+            get
+            {
+                if (IsBooleanType)
+                {
+                    return _field.Value == "true" ? AppResources.Enabled : AppResources.Disabled;
```

Comment:
instead of comparing to that hardcoded string I'd use parse:
```c#
return bool.TryParse(_field.Value, out var b) && b ? AppResources.Enabled : AppResources.Disabled;
```